docs/circuit_diagrams/module_IRTester/README.md

```diff
@@ -0,0 +1,8 @@
+Fritzing Circuit Diagrams
+=========================
+
+This is a diagram of the testing module for the IR sensor and remote. The purpose of this module is to print the output of the IR remote so that specifically mapped buttons on the remote may be implemented in the base project. The remote will be used as an arming button to replace the analogue button currently implemented.
+
+The sensor used has a 3 pin configuration. The (G) pin is used as ground, the (R) pin is used for power and plugged into the 5 volt port and the (Y) pin is used as the sensor and in this instance, run to the 11th digital pin. The Y pin returns the values of specific buttons used on the remote.
+
+Please consult manufacturer specificatitions for your specific pin-layout.
```

Comment:
Small typo here on "specifications". Just fix that on your next commit once you have more IR related code changes.